add documention on grafana backups (#2558)
giantswarm/docs: src/content/tutorials/observability/data-exploration/creating-custom-dashboards/_index.md

## Creating your own dashboard

> To create your own dashboard, you can directly use the Grafana UI and save it there. As we are operating a database for Grafana, dashboards created and saved directly from the UI will be accessible even in case of disaster as long as a backup has been made after the dashboard creation and before the said disaster. For more information on this, please check the related grafana backup [documentation page](https://docs.giantswarm.io/tutorials/observability/data-exploration/grafana-backups/)

Reviewer: But do not both ways coexist? I guess some cases ppl want to create using UI but there may be other reasons to create it in code and enable GitOps. Is still possible? that case I would leave both

> To create your own dashboard, you can directly use the Grafana UI and save it there. As we are operating a database to save grafana data, dashboards created and saved directly from the UI will be accessible even in case of disaster as long as a database backup has been made bewteen the dashboard creation and the said disaster. For more information on this, please check the related grafana backup [documentation page](https://docs.giantswarm.io/tutorials/observability/data-exploration/grafana-backups/)

Author: Yeah I'll add back the other method but concerning the UI one, I think it's straightforward enoguh for not having to add screenshots

## Creating your own dashboard

> You can create a dashboard either with our recommended GitOps approach, or through the Grafana UI. Following you can learn more about both ways. 

Reviewer: ```suggestion
You can create a dashboard either with our recommended GitOps approach, or through the Grafana UI. Following you can learn more about both ways.
```

## Creating your own dashboard

There are two possible ways to create your own dashboard :

> 

Reviewer 2: I still think it would make sense to explain why gitops is better.

Also, the example does not really explain why this annotation is about:
    ## Define the organization in Grafana where the dashboard will be added
    observability.giantswarm.io/organization: Customer

Can you add that this must be the display name of a Grafana Organization CR? We probably should link to the Grafana Organization page as well

> 

Author: I applied your suggestions :)

  namespace: my-namespace
```

> - You can directly use the Grafana UI and save it there. As we are operating a database for Grafana, dashboards created and saved directly from the UI will be accessible even in case of disaster as long as a backup has been made after the dashboard creation and before the said disaster.

Reviewer 2: I think it's hourly backups right?

WDYT about?
```suggestion
- You can also create your dashboards directly through the Grafana UI. Grafana data is stored in a postgresql database which is backed up hourly so we can recover the state of grafana in case of a disaster.
```

  namespace: my-namespace
```

> __warning__: the `observability.giantswarm.io/organization` annotation's value must be equal to an existing GrafanaOrganization CR's display name. For more information on grafana organizations, check the [related page](https://docs.giantswarm.io/tutorials/observability/multi-tenancy/creating-grafana-organization/).

Reviewer 2: ```suggestion
__warning__: the `observability.giantswarm.io/organization` annotation's value must be equal to an existing GrafanaOrganization CR's display name. For more information on Grafana organizations, check the [related page](https://docs.giantswarm.io/tutorials/observability/multi-tenancy/creating-grafana-organization/).
```

> __warning__: the `observability.giantswarm.io/organization` annotation's value must be equal to an existing GrafanaOrganization CR's display name. For more information on Grafana organizations, check the [related page](https://docs.giantswarm.io/tutorials/observability/multi-tenancy/creating-grafana-organization/).

Reviewer 3: ```suggestion
__warning__: the `observability.giantswarm.io/organization` annotation's value must be equal to an existing GrafanaOrganization CR's display name. For more information on Grafana organizations, check [our documentation on multi-tenancy](https://docs.giantswarm.io/tutorials/observability/multi-tenancy/).
```

> This is the preferred approach as it allows shared dashboards across all installations (if you have multiple ones) and comes with a linting job in the CI pipeline to ensure the dahboard is valid.

Reviewer 2: ```suggestion
This is the preferred approach as it allows shared dashboards across all installations (if you have multiple ones) and comes with a linting job in the CI pipeline to ensure the dashboard is valid.
```

### Grafana UI

> You can also create your dashboards directly through the Grafana UI. Grafana data is stored in a postgresql database which is backed up hourly so we can recover the state of grafana in case of a disaster.

Reviewer 2: ```suggestion
You can also create your dashboards directly through the Grafana UI. Grafana data is stored in a Postgresql database which is backed up hourly so we can recover the state of grafana in case of a disaster.
```

> ## Limitations

Reviewer 3: You have a h2 for limitations here, but an h3 limitations above - is the one above a duplication and you can pull that content down here, or do you maybe want to make the limitations an h4 within the h3 of GitOps and Grafana UI? 

But reading the limitations here, I feel like that belongs to the GitOps approach and not at all to the Grafana UI way, because Grafana automatically creates a UID, right?

Author: I removed the `Limitations` subsection and moved its content to the warning part in the GitOps section.

  namespace: my-namespace
```

__warning__: the `observability.giantswarm.io/organization` annotation's value must be equal to an existing GrafanaOrganization CR's display name. For more information on Grafana organizations, check [our documentation on multi-tenancy](https://docs.giantswarm.io/tutorials/observability/multi-tenancy/).

This is the preferred approach as it allows shared dashboards across all installations (if you have multiple ones) and comes with a linting job in the CI pipeline to ensure the dashboard is valid.

### Grafana UI

> You can also create your dashboards directly through the Grafana UI. You can learn more about [how to create a dashboard](https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/create-dashboard/) through Grafana in the official documentation. 

Reviewer: ```suggestion
You can also create your dashboards directly through the Grafana UI. You can learn more about [how to create a dashboard](https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/create-dashboard/) through Grafana in the official documentation.
```
